src/components/Home.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <section>
+        <h2>{"{ CYF Quiz }"}</h2>
+        <p>Hello Coder !<br />
+        Are you ready for the next challenge ?<br />
+        Take a quiz !</p>
+        <ul className="list-unstyled">
+          <li className="mb-3"><Button href="Quiz">HTML</Button></li>
+          <li className="mb-3"><Button href="Quiz">CSS</Button></li>
+          <li className="mb-3"><Button href="Quiz">JavaScript</Button></li>
```

Comment:
These buttons look quite far apart to me — maybe we could make the distance between them a bit smaller? Also, best practice URLs are lowercase, so there should not be a capital `Q`

Reply:
I will change it to 2, I will change the URL to lowercase, that's really good to know.
Thanks a lot for having the time for your feedback, it's very useful.